app/src/main/res/layout/app_bar.xml

```diff
@@ -97,7 +97,7 @@
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:background="?attr/colorPrimary" />
+        android:background="?attr/white" />
```

Comment:
Same issue as above, this parameter value can not be found during the build and will fail the build.  Just leave the original value unchanged